packages/kbn-config-schema/index.ts

```diff
@@ -110,92 +121,42 @@ function ip(options?: IpOptions): Type<string> {
 /**
  * Create an optional type
  */
-function maybe<V>(type: Type<V>): Type<V | undefined> {
+function maybe<V, R>(type: Type<V, R>): Type<V | undefined, R | undefined> {
```

Comment:
This change affects all container types (i.e., types that take other types as arguments)